ui/src/directives/GoBack.js

```diff
@@ -13,7 +13,7 @@ export default {
     }
     else {
       ctx.goBack = () => {
-        vnode.context.$router.replace(ctx.value)
+        vnode.context.$router.go(ctx.single ? -1 : ctx.value)
```

Comment:
$router.go() receives only a Number as param, while ctx.value is a String. So this generates error if not using the `single` modifier.

Reply:
made a new commit, checking for the single to do the go function, or the push function.

I've recheck the vue router docs, and if we use the replace, we will replace the history on the browser, but I don't think it's a good approach, but if you think the replace it's better, can be changed.